playground/frontend/playground_components/test/tools/extract_symbols_java/sdk_mock/Class1.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package sdk_mock;
+
+public class Class1 {
+    public int field11;
+    public String field12;
+    private int field13;
```

Comment:
It's easier to read the test expectation if field names reflect visibility. Please test all 4 visibilities:
```suggestion
    int defaultField;
    public int publicField;
    protected int protectedField;
    private int privateField;
```

Reply:
done